src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -745,7 +776,7 @@ impl WorkunitStore {
   }
 
   pub fn setup_for_tests() -> (WorkunitStore, RunningWorkunit) {
-    let store = WorkunitStore::new(false, Level::Debug);
+    let store = WorkunitStore::new(false, Level::Trace);
```

Comment:
why?

Reply:
Because this change: https://github.com/pantsbuild/pants/pull/14856/files#diff-cb899b178835b1e2138b163355f0e525762e4104d1240e98894172300b87ce8eL858 ... would otherwise cause workunits which are consumed by some tests to be disabled. I thought about adjusting the tests instead, but that felt like it could wait for an actual need.